packages/react/src/errorboundary.tsx

```diff
@@ -0,0 +1,124 @@
+import * as Sentry from '@sentry/browser';
+import * as hoistNonReactStatic from 'hoist-non-react-statics';
+import * as React from 'react';
+
+export const UNKNOWN_COMPONENT = 'unknown';
+
+export type ErrorBoundaryProps = {
+  showDialog?: boolean;
+  dialogOptions?: Sentry.ReportDialogOptions;
+  fallback?: React.ReactNode;
+  renderKey?: string | number;
+  fallbackRender?(fallback: {
+    error: Error | null;
+    componentStack: string | null;
+    resetError(): void;
+  }): React.ReactNode;
```

Comment:
Might be biased because of our app codebase, but I generally see `render*` used as the prefix -- it also makes sense to be consistent with the `fallback` prop. Should we overload `fallback`?

Reply:
That makes sense to me, I'll overload `fallback`.